src/main/java/org/haffson/adventofcode/days/day02/Day02.java

```diff
@@ -0,0 +1,177 @@
+package org.haffson.adventofcode.days.day02;
+
+import org.haffson.adventofcode.ProblemStatusEnum;
+import org.haffson.adventofcode.days.Days;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Scanner;
+
+
+
+/**
+ * Implementation for <i>Day 1: Chronal Calibration</i>.
+ */
+@Component
+public class Day02 implements Days {
+
+    private final String path = "/Users/jenni/dedica/AdventOfCode/Jenni/Day02/src/input.txt";
```

Comment:
This path is an absolute path and no one else will be able to execute this code, without changing the path ;)

Try placing the input into the `src/main/resources` folder. Here are some ways of accessing the file from that folder: https://www.baeldung.com/spring-classpath-file-access

Also, @Matthimatiker will have an opinion on what works best. Try asking him for advice e.g. in the Chat :)